liferay-gradle/blade.servicebuilder.svc/build.gradle

```diff
@@ -23,4 +23,28 @@ buildService {
 	osgiModule = true
 	propsUtil = "com.liferay.blade.samples.servicebuilder.service.util.PropsUtil"
 	testDir = "src/java/test"
+}
+
+startTestableTomcat {
+	checkTimeout = 10 * 60 * 1000
+}
+
+testIntegration {
+	outputs.upToDateWhen { false }
+
+	dependsOn jar, ":initBundle", startTestableTomcat
+	dependsOn ":blade.servicebuilder.api:deploy"
```

Comment:
We don't need to both deploy this and then later install it with JMXDeployer.  Lets just use the JMX deployer for now and not the gradle deploy.